docs/reference/method/MongoDBGridFSBucket-openUploadStream.txt

```diff
@@ -40,7 +40,25 @@ Behavior
 Chunk documents will be created as data is written to the writable stream. The
 metadata document will be created when the writable stream is closed.
 
-.. todo: add examples
+Examples
+--------
+
+.. code-block:: php
+
+   <?php
+
+   $bucket = (new MongoDB\Client)->test->selectGridFSBucket();
+
+   $stream = $bucket->openUploadStream('filename');
+   fwrite($stream, 'foobar');
+   fclose($stream);
+
+   $stream = $bucket->openDownloadStreamByName('filename');
+   var_dump(stream_get_contents($stream, -1, 0));
```

Comment:
The second and third arguments to `stream_get_contents()` shouldn't be necessary, as they default to reading the entirety of the stream with no seeking.